lte/gateway/c/oai/tasks/nas/emm/SecurityModeControl.c

```diff
@@ -449,6 +492,27 @@ int emm_proc_security_mode_complete(
       imeisv.u.num.svn1   = imeisvmob->svn1;
       imeisv.u.num.svn2   = imeisvmob->svn2;
       imeisv.u.num.parity = imeisvmob->oddeven;
+      // Convert to string
+      char imeisv_str[MAX_IMEISV_SIZE + 1] = {0};
+      char imei_tmp[MAX_IMEI_SIZE]         = {0};
+      IMEISV_TO_STRING(&imeisv, imeisv_str, MAX_IMEISV_SIZE + 1);
+      // Copy only 14 digits for validatation.
+      memcpy(imei_tmp, imeisv_str, (MAX_IMEI_SIZE - 1));
```

Comment:
no need to define `imeisv_str`; define `imei_str` and copy 14 bytes from `imeisv` plus null. E.g., 
`IMEISV_TO_STRING(&imeisv, imei_str, MAX_IMEI_SIZE);`

Reply:
imei_tmp will be eliminated by this way...